profiles/common/modules/features/nexteuropa_feature_set/nexteuropa_feature_set.info

```diff
@@ -0,0 +1,49 @@
+name = Nexteuropa Feature set
+description = A module that package some configuration for feature set uses
+core = 7.x
+package = NextEuropa
+dependencies[] = features
+dependencies[] = multisite_drupal_features_set_standard
+features[features_api][] = api:2
+features[user_permission][] = manage feature contact_form
```

Comment:
Should we provide this permission implementation in the scope of the NEPT-182?
This looks more like a task for a separate ticket. Correct me if I'm wrong.